index.js

```diff
@@ -332,6 +334,7 @@ var Util = function(opts) {
             _.token()
             .then(token => API[meth + 'Collection'](name, token, obj))
             .then(res => {
+              if (opts.trace) logfile.write(JSON.stringify(res, null, 0)+'\n')
```

Comment:
A date in ISO format would help ping support with the correct time in case of errors

Reply:
adding it to the start to line.
eg. 
[2018-03-29T06:45:14.890Z]  <res or err here>